datasets/bbc_hindi_nli/bbc_hindi_nli.py

```diff
@@ -0,0 +1,157 @@
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://github.com/avinsit123/hindi-nli-data"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = """
+MIT License
+
+Copyright (c) 2019 MIDAS, IIIT Delhi
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+_TRAIN_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/avinsit123/hindi-nli-data/master/Textual_Entailment/BBC/BBC_recasted_train.tsv"
+)
+_VALID_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/avinsit123/hindi-nli-data/master/Textual_Entailment/BBC/BBC_recasted_dev.tsv"
+)
+_TEST_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/avinsit123/hindi-nli-data/master/Textual_Entailment/BBC/BBC_recasted_test.tsv"
+)
+
+
+class bbc_hindi_nli_config(datasets.BuilderConfig):
+    """BuilderConfig for BBC Hindi NLI Config"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for BBC Hindi NLI Config.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(bbc_hindi_nli_config, self).__init__(**kwargs)
+
+
+class bbc_hindi_nli(datasets.GeneratorBasedBuilder):
+    """BBC Hindi NLI dataset -- Dataset providing textual-entailment pairs for NLI tasks in Hindi"""
+
+    BUILDER_CONFIGS = [
+        bbc_hindi_nli_config(
+            name="bbc hindi nli",
+            version=datasets.Version("1.1.0"),
+            description="BBC Hindi NLI: Natural Language Inference Dataset in Hindi",
+        ),
+    ]
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.Value("string"),
+                    "topic": datasets.Value("string"),
```

Comment:
```suggestion
                    "label": datasets.ClassLabel(names=["neutral", "entailment", "contradiction"]),
                    "topic": datasets.ClassLabel(names=["india", "pakistan", "news", ...]),
```
Could you use the ClassLabel feature type here please ?

You'll need to regenerate the dataset_infos.json after this change

Reply:
I just noticed that there is only entailment and not-entailment.
So I would suggest using 
```
"label": datasets.ClassLabel(names=["not-entailment", "entailment"]),
```
let me know what you think